trustpinning/certs_test.go

```diff
@@ -700,27 +556,53 @@ func filestoreWithTwoCerts(t *testing.T, gun, keyAlg string) (
 
 		certificates[i] = cert
 	}
-	return tempBaseDir, certStore, cryptoService, certificates
+	return tempBaseDir, cryptoService, certificates
 }
 
 func testValidateSuccessfulRootRotation(t *testing.T, keyAlg, rootKeyType string) {
 	// The gun to test
 	gun := "docker.com/notary"
 
-	tempBaseDir, certStore, cs, certificates := filestoreWithTwoCerts(t, gun, keyAlg)
+	tempBaseDir, cs, certificates := generateTwoCerts(t, gun, keyAlg)
```

Comment:
Non-blocking - since we don't care about getting the certs out anymore, `testutils.CreateKey` will generate x509 keys for root keys.  Not sure, but I don't think that would cause an import cycle.